sim/midas/src/main/scala/midas/passes/ElaborateChiselSubCircuit.scala

```diff
@@ -0,0 +1,45 @@
+// See LICENSE for license details.
+
+package midas.passes
+
+import firrtl.{AnnotationSeq, Transform}
+import firrtl.ir.Circuit
+import firrtl.options.{Dependency, PhaseManager}
+import firrtl.stage.{FirrtlCircuitAnnotation, RunFirrtlTransformAnnotation}
+import chisel3.{RawModule}
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselCircuitAnnotation}
+import chisel3.experimental.RunFirrtlTransform
+
+object ElaborateChiselSubCircuit {
```

Comment:
I wrote this to replace the flow in SimulationMapping so that FileAnnotations generated from Chisel in the simulator wrapper would get properly serialized out.